src/app/components/select-field/index.html

```diff
@@ -63,6 +63,13 @@
     >
     Specifies the selection mode that determines whether users can select one item or multiple items in the select field component's modal. The valid options are <sky-code>single</sky-code>, which displays a text input and allows users to select one item, and <sky-code>multiple</sky-code>, which displays a button and allows users to select multiple items.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="showAddNewRecordButton"
+      isOptional="true"
+      defaultValue="false"
+    >
+    Indicates whether to display a button with a <sky-code>New</sky-code> text label. Users can add new items directly from the select field modal. This property accepts <sky-code>boolean</sky-code> values.
```

Comment:
I don't think the code snippet style for the text label follows our general style. I think we generally just refer to buttons generically with no formatting. Also, what do you think of combining the first two sentences to tighten up? For example: "Indicates whether to display a button for users to add new items directly from the select field modal."